src/components/Table/TableRow.tsx

```diff
@@ -91,19 +91,38 @@ export class TableRow<T> extends React.PureComponent<TableRowProps<T>, {}> {
 						typeof column.cellAttributes === 'function'
 							? column.cellAttributes(data, get(data, column.field))
 							: column.cellAttributes || {};
-					return (
-						<a
-							{...attributes}
-							href={href}
-							data-display="table-cell"
-							data-key={keyAttribute}
-							onClick={this.props.onRowClick}
-							{...cellAttributes}
-							key={column.key || (column.field as string)}
-						>
-							{renderField(data, column)}
-						</a>
-					);
+					try {
+						const url = new URL(href ?? '');
+						return (
+							<a
+								{...attributes}
+								data-display="table-cell"
+								data-key={keyAttribute}
+								onClick={(props) => {
+									this.props.onRowClick(props);
+									window.open(url.toString(), '_blank');
+								}}
+								{...cellAttributes}
+								key={column.key || (column.field as string)}
+							>
+								{renderField(data, column)}
+							</a>
```

Comment:
How about parsing the url in its own try-catch (check if there is one first), and then have a single render which sets the correct properties (rather than repeating it).

Reply:
I happened to do that right after creating the PR, thought I pushed the code but I guess not, my bad 😓